Assets/Scripts/Game/UserInterfaceWindows/DaggerfallGuildServiceDonation.cs

```diff
@@ -105,7 +105,7 @@ public GuildServiceDonationMacroDataSource(DaggerfallGuildServiceDonation guildS
 
             public override string God()
             {
-                return Temple.GetDivine(parent.BuildingFactionId).ToString();
+                return Temple.GetDivineLocalized(parent.BuildingFactionId).ToString();
```

Comment:
The ".ToString()" can be removed, `GetDivineLocalized` already returns a string

Reply:
Fixed